requirements.txt

```diff
@@ -1,12 +1,12 @@
 # required scientific packages
-numpy>=1.14.0
+numpy>=1.16.2
```

Comment:
The lgtm tool is not parsing this `requirements.txt` file the way it should. This leads to a failure to install the python dependencies. Here is a snippet from the [logfile `fef813e`](https://lgtm.com/projects/g/espressomd/espresso/logs/rev/pr-9f763a60debebfb03a63d2029ecdf53bcd8ee7af/lang:python/stage:Build%20python_fef813e78b52f4767dfbc12642a627d8773bc2cc):
```
[2021-07-06 07:55:01] [build-stdout] Installing scipy==0.*,>=0.19.0 with dependencies.
[2021-07-06 07:55:01] [build-stdout] Calling install -r /tmp/semmle-requirementsJndGWr.txt
[2021-07-06 07:55:01] [build-stdout] Calling /opt/work/lgtm-workspace/venv/bin/python -m pip install -r /tmp/semmle-requirementsJndGWr.txt
[2021-07-06 07:55:02] [build-stdout] Collecting scipy==0.*,>=0.19.0
[2021-07-06 07:55:02] [build-stdout]   Downloading scipy-0.19.1.tar.gz (14.1 MB)
[2021-07-06 07:55:04] [build-stdout] Using legacy 'setup.py install' for scipy, since package 'wheel' is not installed.
[2021-07-06 07:55:04] [build-stdout] Installing collected packages: scipy
[2021-07-06 07:55:04] [build-stdout]     Running setup.py install for scipy: started
[2021-07-06 07:55:12] [build-stdout]     Running setup.py install for scipy: finished with status 'error'
```
The rule `scipy==0.*` is a mistake. By bumping scipy rule to `scipy>=1.1.0`, lgtm introduced the rule `scipy==1.*` which allowed it to successfully install scipy. The numpy was updated to match the scipy version. I was going to bump these versions soon in a docker image update anyway.

The lgtm tool is still complaining about the vtk dependency ([logfile `c88762d`](https://lgtm.com/projects/g/espressomd/espresso/logs/rev/pr-1959c9a610379acc52c1193fd8e151dfb9896464/lang:python/stage:Build%20merge_c88762d4a00a2d9d45cb72e1bd3fa1fea74c59ef)), but it managed to work around it using a python wheel archive. The lgtm analysis still timed out, but I'm not sure why, and the lgtm logfiles don't report any fatal error.